frontend/templates/troubleshooting/index.tsx

```diff
@@ -70,14 +61,22 @@ function NavBar({ editUrl }: { editUrl: string }) {
          justify="center"
       >
          <Flex maxW="1280px" flexGrow="1">
-            <Box flexGrow="1">Breadcrumbs</Box>
-            <Box>Parts/Guide/Answers</Box>
+            <Breadcrumbs />
+            <AreaLinks />
```

Comment:
Are we going to insert the text content in `Breadcrumbs` and `AreaLinks` in the future?

Reply:
It probably won't be just text content, but yes, we plan to have both those components eventually.